patches/wikibase-sdk+7.9.1.patch

```diff
@@ -0,0 +1,73 @@
+diff --git a/node_modules/wikibase-sdk/lib/queries/get_entities_from_sitelinks.js b/node_modules/wikibase-sdk/lib/queries/get_entities_from_sitelinks.js
```

Comment:
Do we need this file?

Reply:
No, it's just my modification to the library before I stopped using it.

Reply:
Its okay if I remove it?

Reply:
Yep